backend/src/services/ai/chat-completion.service.ts

```
@@ -378,7 +386,10 @@ export class ChatCompletionService {
 
       // Send request with upstream error mapping
       const { result: stream } = await this.openRouterProvider.sendRequest((client) =>
-        client.chat.completions.create(request as OpenAI.Chat.ChatCompletionCreateParamsStreaming)
+        client.chat.completions.create(
+          request as OpenAI.Chat.ChatCompletionCreateParamsStreaming,
```

ESLint

[error] Replace `⏎··········request·as·OpenAI.Chat.ChatCompletionCreateParamsStreaming,` with `request·as·OpenAI.Chat.ChatCompletionCreateParamsStreaming,·{`

(prettier/prettier)